docs/catalog/cloud/use-circuit-breaker.md

```diff
@@ -0,0 +1,43 @@
+---
+version: 1.0
+submitted_by: markus-ntt-seidl
+published_date: TBD
+category: cloud
+description: [PATTERN_DESCRIPTION_METADATA]
+tags: 
+ - compute
+ - size:medium
+---
+
+# Use circuit breaker patterns
+
+## Description
+
+Modern applications need to communicate with other applications on a regular basis. These other applications have their own deployment schedule, downtimes and availability or the network connection to these application might have problems.
+If the other application is not reachable, all network requests against this other application will fail and future network requests are futile.
+
+## Solution
+
+Circuit breaker patterns reject the request to the other application if health checks against the other application have failed to complete successfully. In this case a lot of network requests can be saved. 
```

Comment:
again, i think it might be good to reference Azure's documentation and include something more aligned to the following:
`The Circuit Breaker pattern can prevent an application from repeatedly trying to execute an operation that's likely to fail. Allowing it to continue without waiting for the fault to be fixed or wasting CPU cycles while it determines that the fault is long lasting. The Circuit Breaker pattern also enables an application to detect whether the fault has been resolved. If the problem appears to have been fixed, the application can try to invoke the operation.`

I think it would also be good to include in the solution why this helps reduce energy usage.

Reply:
I included a small reference that network requests _and_ energy can be saved.
Otherwise the patterns v1 is always a short solution/from and the Azure one just seems to be expanded that network requests are CPU cycles.

I really do not see the benefit for patterns-v1. I think that could be something for patterns-v2 for sure.